src/blog/2024/07/customise-theming-in-your-dashboards.md

```diff
@@ -0,0 +1,235 @@
+_Screenshot showing the header style options in the dashboard settings_
+
+The following options are available for header styling:
+
+### Default
+
+This option as it name suggest it is the default option set for header. In which the header will get hidden if we scrolled down.
+
+!["Image showing the dashboard with default header"](./images/default-header.gif "Image showing the dashboard with default header"){data-zoomable}
+_Image showing the dashboard with default header_
+
+### Hidden
+
+Selecting this option completely hides the header, allowing you to use that space for other purposes.
+
+!["Image showing the dashboard with hidden header"](./images/hidden-header.png "Image showing the dashboard with hidden header"){data-zoomable}
+_Image showing the dashboard with hidden header_
+
+### Fixed
+
+Selecting this option keeps the header fixed at the top. This means that when you scroll the page down, the header will remain visible.
+
+!["Image showing the dashboard with fixed header"](./images/fixed-header.gif "Image showing the dashboard with fixed header"){data-zoomable}
+_Image showing the dashboard with fixed header_
+
+## Changing Dashboard Theme
+
+In this section of the guide, you will learn how to change the Dashboard theme, where you can adjust the colors of the header, navigation sidebar, group and page backgrounds, the border color of groups, and the padding, sizing, and gaps between pages, groups, and widgets.
+
+To edit the existing theme:
+
+1. Go to the Dashboard 2.0 sidebar.
+2. Switch to the "Theme" tab.
+
+!["Screenshot showing the dashboard theme tab in the sidebar"](./images/dashboard-theme-tab.png "Screenshot showing the dashboard theme tab in the sidebar"){data-zoomable}
+_Screenshot showing the dashboard theme tab in the sidebar_
+
+3. Click on the edit button next to the theme.
+4. You can adjust the header color and the primary color (which applies to the navigation sidebar and elements like buttons and dropdowns) under the "Primary" section. In the "Pages" section, set the background color for pages, and in the "Groups" section, adjust the background color and border color of groups.
+
+!["Screenshot showing the theme properties dialog"](./images/dashboard-theme-tab.png "Screenshot showing the theme properties dialog"){data-zoomable}
+_Screenshot showing the theme properties dialog_
+
+5. Under "Sizing," adjust the page padding (the space between dashboard groups), the page border, group gap, group border radius (the thickness of the group border), and widget gap.
+
+For more information on  theme, how to add new themes, and set themes for pages, refer to the [Comprehensive guide: Node-RED Dashboard 2.0 layout, sidebar, and styling](/blog/2024/05/node-red-dashboard-2-layout-navigation-styling/#understanding-dashboard-2.0-theme). Additionally, this guide covers Node-RED Dashboard 2.0 layouts, themes, and custom styling in detail.
```

Comment:
@sumitshinde-84 I noticed that you've already added a link to that article as part of the content, but it should also appear as related content in the right column. Could you please check the tags? [Here's how they work](https://flowfuse.com/handbook/customer/marketing/blog/#custom-tags:~:text=Keep%20in%20mind%20that%20for%20an%20article%20to%20be%20considered%20related%2C%20the%20majority%20of%20tags%20must%20match%2C%20with%20allowance%20for%20a%20difference%20in%20one%20tag.) for suggesting content.